### PR TITLE
fix(Configuration & Maintenance): NEPT-414: reinject NEPT-60 fix

### PR DESCRIPTION
**Pull request by gillesdeudon**

### CODE REVIEW

File: resources/multisite_drupal_standard.make [hunk 1]
```diff
@@ -617,6 +617,11 @@ projects[tmgmt][subdir] = contrib
 projects[tmgmt][patch][] = https://www.drupal.org/files/issues/support_for_link_field-2489134-9.patch
 ; @see https://www.drupal.org/node/272245
 projects[tmgmt][patch][] = https://www.drupal.org/files/issues/tmgmt-test_translator_missing-2722455-2.patch
+; NEPT-60
+; @see https://www.drupal.org/node/2812863
+; This patch fix the insufficient access check on Views.
```
Comment (voidtek) [inline resources/multisite_drupal_standard.make]: The syntax is not good.

; #2812863 : Insufficient access check on Views
; https://www.drupal.org/node/2812863
; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-60

**Comment by voidtek** [comment]
:+1:

**Comment by jfhovinne** [comment]
:+1:
Fine, let us wait for the CI tools green lights as well.

**Comment by jfhovinne** [comment]
OK for the merge.

**Comment by jfhovinne** [comment]
:+1:

**Pull request merged**